Platon/backend/app/search_engine/helpers.py

```diff
@@ -101,6 +101,10 @@ def remove_punctuation(search_query):
     def sort_ids(related_id_score_list):
         return sorted(related_id_score_list, key=lambda tup: tup[1])
 
+    @staticmethod
+    def sort_results(results_list, key_list, reverse):
+        return sorted(results_list, key=lambda x: " ".join([x[key].lower() for key in key_list]), reverse=reverse)
+
```

Comment:
Why did you change the function that I implemented? I used this function on user search to sort the results by using their alphabetical order. If you changed this function, my code will not be able to sort the users according to alphabetical order.